### DISCUSSION TITLE
Denied to use Actions

### DISCUSSION THREAD

**Discussion by 1pipinto** [community/community#52260] [Actions]
### Select Topic Area

Bug

### Body

I don't understand why I'm being denied using the "actions" function
![Screenshot_20230407-110138](https://user-images.githubusercontent.com/111198367/230621902-d934d6fc-18c3-47cd-a171-8b3e611d7070.jpg)

**Comment by jackieton** [comment]
Check if "Read and write permissions" are enabled in Settings.

**Reply by 1pipinto** [reply]
It used to work but now it doesn't work anymore but it's active

**Comment by airtower-luna** [comment]
Maybe some of your workflows triggered an abuse detection mechanism. But that's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by 1pipinto** [reply]
I don't know if this is some kind of abuse, but I used a code to get a vps via ngrok, would that be abuse?

**Reply by airtower-luna** [reply]
Sounds to me like it'd match at least two categories listed as prohibited in the Actions terms: https://docs.github.com/en/site-policy/github-terms/github-terms-for-additional-products-and-features#actions

**Reply by 1pipinto** [reply]
So, in short, the method I used was a kind of abuse?, because I wanted a virtual machine just for testing :(

**Reply by airtower-luna** [reply]
That's what I *assume*, yes. You'll have to ask GitHub support for a definite statement, and only they can restore Actions for you.
